proc/breakpoints.go

```diff
@@ -33,7 +33,18 @@ type Breakpoint struct {
 	HitCount      map[int]uint64 // Number of times a breakpoint has been reached in a certain goroutine
 	TotalHitCount uint64         // Number of times a breakpoint has been reached
 
-	Cond ast.Expr // When Cond is not nil the breakpoint will be triggered only if evaluating Cond returns true
+	// When DeferCond is set the breakpoint will only trigger
+	// if the caller is runtime.gopanic or if the return address
+	// is in the DeferReturns array.
+	// Next sets DeferCond for the breakpoint it sets on the
+	// deferred function, DeferReturns is populated with the
+	// addresses of calls to runtime.deferreturn in the current
+	// function. This insures that the breakpoint on the deferred
+	// function only triggers on panic or on the defer call to
+	// the function, not when the function is called directly
+	DeferCond    bool
+	DeferReturns []uint64
```

Comment:
From what I understand from reading https://github.com/golang/go/blob/master/src/runtime/panic.go#L269 is that there is only ever one call to `runtime.deferreturn` and, when jumping into the deferred function, it is called such that it returns not into `runtime.deferreturn` but into the caller of `runtime.deferreturn`, at the point right before it was called, essentially looping until all deferred functions are called.

As such, we shouldn't need a slice, only a single location, correct?